{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/api/serializers.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import  serializers
+from {{ cookiecutter.project_slug }}.users.models import User
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = '__all__'
```

Comment:
This looks a bit too open ended. Can we lock it down a bit?

Reply:
Hi. What does it mean open ended? Should I restrict the fields? Thanks!

Reply:
Sorry, the choice of words wasn't the best. Yes, I meant that we should restrict the fields, this is exposing too much information.